pkg/pb/gc.proto

```diff
@@ -0,0 +1,28 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+syntax = "proto3";
+option go_package = "pb";
+
+package gc;
+
+import "google/protobuf/timestamp.proto";
+
+service GarbageCollection {
+    rpc Delete(DeleteRequest) returns (DeleteResponse);
+}
+
+message BloomFilter {
+    uint32 seed = 1;
+    uint32 k = 2;
+    uint32 bits_per_element = 3;
+    bytes table = 4;
+}
+
+message DeleteRequest {
+    google.protobuf.Timestamp creation_date = 1;
```

Comment:
Add `[(gogoproto.stdtime) = true]` like here: https://github.com/storj/storj/blob/15e74c8c3d68821eac13f4d2cccf89ba4604b237/pkg/macaroon/types.proto#L27

This will set the type to `time.Time` in the generated Go code, which is more convenient and we won't need to convert from Timestamp to Time.